base/db/R/get.new.site.R

```diff
@@ -0,0 +1,121 @@
+##' Get new site info using provided site information
+##'
+##' @title Get New Site Info
+##' @param site a dataframe with site information including id, lat, lon, and time_zone.
+##' @param con Database connection object.
+##' @return a dataframe with new site information on lat, lon and time_zone
+##' @export
+##' @author Abhinav Pandey
+##' @importFrom digest digest
```

Comment:
What does this do? First, as far as I can tell this function is never used here. Second, where it is used I don't see why it can't be called by namespace